src/Blazor.Auth0.ClientSide/Blazor.Auth0.ClientSide.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
The official recommendation is to target the lowest version of .NET Standard possible, and there's no known reason for upgrading it, please revert this change.